requirements.in

```diff
@@ -52,3 +53,4 @@ xlrd==2.0.1  # for reading .xls files (but not .xlsx)
 openpyxl==3.1.0  # for reading .xlsx files (but not .xls)
 zenpy==1.1.3
 jinja2==3.0.3  # This version needed for sphinx to not raise errors
+pip-tools
```

Comment:
I don't mind adding this here - does pip-tools add itself to requirements.txt? I don't see it there

Reply:
I think I commited this accidentally. But since it doesn't appear in requirements.txt not sure if it's useful to have it in `in`at all?

Reply:
I like having it around so that we can easily tell people how to run pip-compile to build the new file. otherwise you need to install it every single time. I'm in favour of making it appear properly if we can do that. maybe you added it after you ran pip-compile?

Reply:
it was a duplicte entry in requirements.in and pip-tools was already in requirements.txt